src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/AuthDetails.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+public class AuthDetails {
+    final UserDetails userDetails;
```

Comment:
can we set an access level on these fields please?
currently the class is public but none of the fields are

Reply:
it is default and public on purpose.
This means we  can create the Object on the SampleData but the information is only visible internally to the package. 
Id prefer just to use a kotlin data object and get rid of this.